### PR TITLE
feat: create rpc method to link to swap/bridge

### PR DESCRIPTION
**Pull request by camerow**
> Try out Leather build ca5b30c — [Extension build](https://github.com/leather-io/extension/actions/runs/12302880200), [Test report](https://leather-io.github.io/playwright-reports/will/update-rpc-methods), [Storybook](https://will/update-rpc-methods--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=will/update-rpc-methods)

### CODE REVIEW

**Comment by fbwoolf** [comment]
I know @pete-watters implemented the `rpcOpen` with the intention (I think) of allowing a specific route in the wallet to be opened? It was implemented initially to just open the home route, but maybe we can just expand that one?

File: src/background/messaging/rpc-message-handler.ts [hunk 1]
```diff
@@ -22,6 +23,10 @@ export async function rpcMessageHandler(message: WalletRequests, port: chrome.ru
       await rpcOpen(message, port);
       break;
     }
+    case 'swap': {
```
Comment (kyranjamie) [inline src/background/messaging/rpc-message-handler.ts:26]: Would call it this message something less terse, to indicate that this isn't our API to initiate/track a swap, rather, just opening that page.

**Comment by pete-watters** [comment]
> I know @pete-watters implemented the `rpcOpen` with the intention (I think) of allowing a specific route in the wallet to be opened? It was implemented initially to just open the home route, but maybe we can just expand that one?

👍 maybe we can do that. We had a follow on task to extend `rpcOpen` [here](https://github.com/leather-io/extension/issues/5821) with some more implementation requests.

**Comment by kyranjamie** [comment]
Is the intention of this to be only for use with Leather? If so I'd suggest we prefix the method with something like `leather_*`

**Pull request closed without merging**